identity-field/README.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: pattern
+title: Identity-Field
+folder: identity-field
+permalink: /patterns/identity-fields/
+categories: Structural
+tags:
+- Data access
+---
```

Comment:
Check out the yaml frontmatter requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute